packages/base/source/server/errors.js

```diff
@@ -12,4 +11,4 @@ Space.Error.extend(Todos, 'TodoNotFoundError', {
     Space.Error.constructor.call(this);
     this.message = `No todo with id ${todoId} found.`;
```

Comment:
@DominikGuzei, please look into this. If I am using Space.Error wrong int his instance we should change it.